lib/utils/format.ts

```diff
@@ -24,3 +24,35 @@ export const formatPostalCode = (postalCode: string): string => {
 export const formatNumberInput = (numberInputString: string): string => {
   return numberInputString.replace(/^\$/, '');
 };
+
+/**
+ * Format date to be in MM/DD/YYYY format and in UTC time zone to avoid the date being set back by a day
+ * @param {Date} date date to be formatted
+ * @returns {string} formatted date
+ */
+export const formatDate = (date: Date): string => {
+  return new Date(date).toLocaleDateString('en-US', { timeZone: 'UTC' });
+};
+
+/**
+ * Format date to be in written in the following format: Sep 11 2021, 03:07 pm
+ * @param {Date} date date to be formatted
+ * @returns {string} formatted verbose date
+ */
+export const formatDateVerbose = (date: Date): string => {
+  const localeDateString = new Date(date).toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'short',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: 'numeric',
+  });
+
+  // Removes first comma and writes am/pm in lowercase
+  const formattedDate =
+    localeDateString.replace(',', '').substring(0, localeDateString.length - 3) +
+    localeDateString[localeDateString.length - 2].toLowerCase() +
+    localeDateString[localeDateString.length - 1].toLowerCase();
+
+  return formattedDate;
+};
```

Comment:
What do you think about having both of these in the same function and adding a boolean parameter: `verbose` 🤔  
Imo having one function would make the code more concise and easier to read

Reply:
Having one function would allow us to keep all the date logic in one place. However, the date formatting logic is quite different in these two util functions. If we need to make changes in the future or maintain the logic, having them decoupled is better as it's slightly easier to isolate issues.